src/transport/SessionManager.cpp

```diff
@@ -445,22 +445,6 @@ void SessionManager::ExpireAllPASESessions()
     });
 }
 
-bool SessionManager::MarkSessionsAsDefunct(const ScopedNodeId & node, const Optional<Transport::SecureSession::Type> & type)
```

Comment:
Is there a particular reason we're removing this function? Seems like it still provides value?

Reply:
I was removing it since I thought it was a little to aggressive for something to mark all session of a particular node as defunct. But if you think it is okay I will keep it as is